src/screens/Menu/Menu.js

```diff
@@ -0,0 +1,298 @@
+        title: 'App settings',
+        emoji: 'gear',
+        card: true,
+        action: () => navigation.navigate(APP_SETTINGS),
+      },
+      {
+        key: 'referFriends',
+        title: 'Refer friends',
+        icon: 'present',
+        iconColor: colors.labelTertiary,
+        action: () => this.toggleSlideModalOpen('referralCode'),
+      },
+      {
+        key: 'community',
+        title: 'Community',
+        icon: 'like',
+        iconColor: colors.labelTertiary,
+        action: () => navigation.navigate(COMMUNITY_SETTINGS),
+      },
+      {
+        key: 'chatWithSupport',
+        title: 'Chat with support',
+        icon: 'help',
+        iconColor: colors.helpIcon,
+        action: () => Intercom.displayMessenger(),
+      },
+      {
+        key: 'knowledgeBase',
+        title: 'Knowledge base',
+        icon: 'dictionary',
+        iconColor: colors.positive,
+        action: () => Intercom.displayHelpCenter(),
+      },
+    ];
+  }
+
+  renderMenuItem = ({ item }) => {
+    const {
+      title,
+      action,
+      labelBadge,
+      card,
+      emoji,
+      icon,
+      iconColor,
+    } = item;
+
+    if (card) {
+      return (
+        <ListCard
+          title={title}
+          action={action}
+          labelBadge={labelBadge}
+          customIcon={<StyledEmoji name={emoji} />}
+        />
+      );
+    }
+
+    return (
+      <SettingsListItem
+        label={title}
+        onPress={action}
+        labelBadge={labelBadge}
+        icon={icon}
+        iconColor={iconColor}
+      />
+    );
+  }
+
+  toggleSlideModalOpen = (modal: ?string = null) => {
+    this.setState({ visibleModal: modal });
+  }
+
+  render() {
+    const items = this.getMenuItems();
+    const { visibleModal } = this.state;
+    const { logoutUser, user, theme } = this.props;
+    const currentTheme = getThemeType(theme);
+    const logo = currentTheme === LIGHT_THEME ? headerLogo : headerLogoDarkMode;
+
+    return (
+      <ContainerWithHeader
+        headerProps={{ leftItems: [{ close: true }], centerItems: [{ custom: <HeaderLogo source={logo} /> }] }}
+      >
+        <FlatList
+          data={items}
+          keyExtractor={item => item.key}
+          renderItem={this.renderMenuItem}
+          contentContainerStyle={{ width: '100%', padding: spacing.layoutSides, paddingBottom: 40 }}
+          ListFooterComponent={
+            <Footer>
+              <LinksSection>
+                <LegalTextLink onPress={() => this.toggleSlideModalOpen('termsOfService')}>
+                 Terms of Use
+                </LegalTextLink>
+                <LegalTextLink>  •  </LegalTextLink>
+                <LegalTextLink onPress={() => this.toggleSlideModalOpen('privacyPolicy')}>
+                 Privacy policy
+                </LegalTextLink>
+              </LinksSection>
+              <LogoutSection>
+                <LogoutIcon name="signout" />
+                <LogoutTextLink onPress={logoutUser}>
```

Comment:
In previous implementation we used to have an alert to double check if user really wants to log out. It's crucial, I think, especially if you haven't backed up your wallet yet. Can we, please, add it here as well?